### DISCUSSION TITLE
Add option for muting interstitial Ads.

### DISCUSSION THREAD

**Discussion by afkcodes** [invertase/react-native-firebase#4451] [Ideas]
Can we have an option to mute the interstitial Ads as for a music app its starts playing and does ruin the experience. I looked into the Android Docs and found this. Its seems possible to mute.

https://developers.google.com/admob/android/global-settings

**Comment by mikehardy** [comment]
Anything is possible if there are underlying APIs we may access on android and ios, but everything will require a PR contributed from the user community, if you want to take it up, the "tests" directory in the repository here has a README which shows you how to get set up to develop + test a feature

**Reply by afkcodes** [reply]
will surely try, though i have very little knowledge of native development and experience in iOS. Android i can still try.

**Reply by mikehardy** [reply]
I am also objectively not very good at Objective-C, however the tests project is set up pretty well so that if you follow existing patterns you can probably succeed. I had never worked with Objective-C before implementing a few things here and I was amazingly (to me) able to get it done. Good luck
